---------------------------------
[2025-05-20 13:45:43][INFO][200]:La conexion [dbconn_standard] esta desabilitada en el archivo .config
---------------------------------
[2025-05-20 13:45:43][INFO][200]:Iniciando ...
---------------------------------
[2025-05-20 13:45:46][INFO][200]:Model loaded in 3.6506507396698 seconds
---------------------------------
[2025-05-20 13:45:46][INFO][200]:[INFO] Reading from rtsp://100.108.97.81:8554/cam4
---------------------------------
[2025-05-20 13:45:46][INFO][200]:Starting multithreading
---------------------------------
[2025-05-20 13:45:46][INFO][200]:[INFO] Starting the capturing process in the camera1
---------------------------------
[2025-05-20 13:45:46][INFO][200]:[INFO] Initialized movement detection object for camera1
---------------------------------
[2025-05-20 13:45:46][INFO][200]:[INFO] Starting human detection for camera1
---------------------------------
[2025-05-20 13:47:01][INFO][200]:[INFO] camera1 : Starting recording
---------------------------------
[2025-05-20 13:47:01][INFO][200]:[INFO] Starting ffmpeg recording to /home/crowdcounting/sophie/pinos-analytics-cv/src/IOT-Agent/records/2025_05_20/camera1/2025_05_20-01_47_01_PM.mp4
---------------------------------
[2025-05-20 13:47:09][EXCEPTION][500]:PYTHON-EXCEPTION:

>>>>>>>>>>>>>>>[TRACEBACK]<<<<<<<<<<<<<<<

  File "/home/crowdcounting/sophie/pinos-analytics-cv/src/IOT-Agent/main.py", line 10, in <module>
    ProjectDirector.go( sys.argv )
  File "/home/crowdcounting/sophie/pinos-analytics-cv/src/IOT-Agent/System/Director/ProjectDirector.py", line 133, in go
    ProjectDirector().setFlux( argv )
  File "/home/crowdcounting/sophie/pinos-analytics-cv/src/IOT-Agent/System/Director/ProjectDirector.py", line 113, in setFlux
    self.__play(
  File "/home/crowdcounting/sophie/pinos-analytics-cv/src/IOT-Agent/System/Director/ProjectDirector.py", line 73, in __play
    time.sleep(DURATION_SECONDS)

>>>>>>>>>>>>>>[ERROR_DESC]<<<<<<<<<<<<<<<

  [KeyboardInterrupt]:

>>>>>>>>>>>>>>>>>>>><<<<<<<<<<<<<<<<<<<<<

